Cargo.toml

```diff
@@ -20,7 +20,7 @@ proc-macro2 = { version = "1", optional = true }
 protobuf = { version = "2", optional = true }
 protobuf-codegen = { version = "2", optional = true }
 grpcio-compiler = { version = ">=0.5.0", default-features = false, optional = true }
-prost-build = { version = "0.6", optional = true }
+prost-build = { version = ">=0.6.0", optional = true }
```

Comment:
This seems like an odd requirement to me: 0.7 can have breaking changes over 0.6, which might prevent this crate building or running correctly